src/event-schemas/dom-event.json

```diff
@@ -16,8 +16,12 @@
         "elementId": {
             "type": "string",
             "description": "DOM element ID."
+        },
+        "cssLocator": {
+            "type": "string",
+            "description": "CSS Locator string."
         }
     },
     "additionalProperties": false,
-    "required": ["version", "event", "elementId"]
+    "required": ["version", "event", "elementId", "cssLocator"]
```

Comment:
We probably want `cssLocator` field to be an optional field. We always provide `elementId`, but we do not always provide `cssLocator`.

Reply:
Removed `cssLocator` from being a required property.